src/store/slicePopularCategory/slicePopularCategories.ts

```diff
@@ -0,0 +1,54 @@
+import {createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import { LOAD_STATUSES } from '../../constants'
+import {Category, Good,Api } from "api/api";
+
+const api= new Api ();
+
+export interface State {
+  popularCategories:{ category: Category; items: Good[] }[];
+  loadStatus: LOAD_STATUSES;
+}
+
+const initialState: State = {
+  popularCategories:[{
```

Comment:
тут тоже пустой массив как начальный стейт 
```suggestion
  popularCategories: [],
```